scripts/yocto-build-and-test.sh

```diff
@@ -581,6 +583,20 @@ build_and_test_client() {
                     --set-version-of mender-gateway-qemu-commercial \
                     --version pr || true
             fi
+
+            if grep mender-image-full-cmdline-rofs-commercial <<<"$images_to_build"; then
+                filename="clean-mender-image-full-cmdline-rofs-commercial-${machine_name}.${extension}"
+
+                $WORKSPACE/meta-mender/meta-mender-qemu/docker/build-docker \
+                    -I "${BUILDDIR}/tmp/deploy/images/${machine_name}/${filename}.gz" \
+                    -i mender-image-full-cmdline-rofs-commercial \
+                    $machine_name \
+                    -t registry.mender.io/mendersoftware/mender-qemu-rofs-commercial:pr
+                # (Review-question) - As long we're not testing this image, this is not needed
```

Comment:
Review answer,

We will test this image later in this epic (soon rather than late), so I suggest we have this code already in place so that at the time of working on the integration tests we only need to modify `integration` repo.